src/errors/utils/dbErrorHandling.js

```diff
@@ -0,0 +1,17 @@
+import AppError from '../AppError.js'; // Update the path as needed
+
+export function applyErrorHandlingMiddleware(schema) {
+  schema.post('save', (error, doc, next) => {
+    if (error) {
+      if (error.name === 'ValidationError') {
+        next(new AppError('Validation error', 400, true));
```

Comment:
try not to hardcode the status codes and use the the constants from the errorManagment object